src/others/contexts/service.ts

```diff
@@ -0,0 +1,15 @@
+export type ID = string;
+
+export enum SERVICE_TYPE {
+  EXTERNAL = 'External',
+  SUPPLIES = 'supplies',
+  CHAT = 'chat',
+  INTERNAL = 'Internal',
+}
+
+export interface Service {
+  id: ID;
+  name: string;
+  type: SERVICE_TYPE;
+  path: string;
+}
```

Comment:
Ah, gotcha. Since that data doesn't exist yet in the spreadsheet, perhaps we should keep this PR as is, and update the logic when we get a new `dataset.csv`?

Reply:
hmm. Can you explain the difference between `Category`, `Type`, and `Origin`, and the possible options we want to support for each?

From what I gather, `Type` is like a level higher than `Category` and used to determine the icon we display. `Origin`, which can only be `Internal` and `External`, is used to determine which section to put the service under. And `Category` is used as the button display text and is one of 9 options:

Request supplies
Entry steps
Housing
Financial Support from the Government
Job Listing platform
Language Learning
Health Care
Legal Support
Elementary Education
Schools
Universities
Community Groups

Maybe we don't need a `Type` Column and we can just have an in-code mapping of `Category` -> `Icon`? It seems like there are only a couple `Categories` that need a special icon - request supplies and the chat bot. I don't think we need a column for that if so. 

Thoughts?

Reply:
cc @gustavoiwanaga.

If the categories are consistent, it'll make translations easier.

Reply:
Answering @jschlenker's question:

Categories will be the same across countries but we should flex based on user's feedback.